pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -1036,23 +1039,25 @@ func handleGlobalConfigModify(ctxArg interface{}, key string,
 		return
 	}
 	log.Infof("handleGlobalConfigModify for %s\n", key)
-	status := cast.CastGlobalConfig(statusArg)
-	var gcp *types.GlobalConfig
+	status := cast.CastConfigItemValueMap(statusArg)
+	var gcp *types.ConfigItemValueMap
 	debug, gcp = agentlog.HandleGlobalConfigNoDefault(ctx.subGlobalConfig,
 		agentName, debugOverride)
 	if gcp != nil {
-		ctx.globalConfig = gcp
+		ctx.configItemValueMap = gcp
 	}
 	foundAgents := make(map[string]bool)
-	if status.DefaultRemoteLogLevel != "" {
-		foundAgents["default"] = true
-		addRemoteMap("default", status.DefaultRemoteLogLevel)
+	if _, ok := status.AgentSettings["logmanager"]; ok {
```

Comment:
same as above.. Don;t need to do 2 lookups:

if as, ok:= status.AgentSettings["logmanager"]; ok {
    if logLevel, ok := as["loglevel"]; ok && logLevel.StringValue != "" {
        foundAgents["default"] = true
        addRemoteMap("default", logLevel.StringValue)
    }
}
}